docs/zetaclient/hotkey_tss_passwords.md

```diff
@@ -0,0 +1,34 @@
+## Hot Key and TSS key-share Passwords
+
+### Zetaclient
+Previously there were two environment variables being used to store passwords encrypting the tss key file and local operator keyring file.
+
+* HOTKEY_PASSWORD
+* TSS_FRAGMENT_SEED
```

Comment:
Need to precise it used to be given as env vars

Reply:
Line 4, I mentioned they were env variables. Is that what you meant?